man/systemd-vmspawn.xml

```diff
@@ -498,6 +498,14 @@
     <title>Environment</title>
 
     <variablelist class='environment-variables'>
+      <varlistentry>
+        <term><varname>$SYSTEMD_VMSPAWN_NETWORK_MAC</varname></term>
```

Comment:
Hmm, is this necessary? If necessary, why not to make this a regular argument? E.g. `--network-tap-mac=` or so.

Reply:
I did it like this to mirror what nspawn supports, I'd be happy to add an argument for it :)

Reply:
Oh, I did not know nspawn supports similar env var.
Then, please document it in docs/ENVIRONMENT.md, and hide it from the man page, at least until we officially expose the one for nspawn.